kornia/geometry/liegroup/so2.py

```diff
@@ -200,24 +202,23 @@ def identity(cls, batch_size: Optional[int] = None, device=None, dtype=None) ->
             >>> s = So2.identity(batch_size=2)
             >>> s
             Parameter containing:
-            tensor([[1.+0.j],
-                    [1.+0.j]], requires_grad=True)
+            tensor([1.+0.j, 1.+0.j], requires_grad=True)
         """
-        real_data = tensor([1.0], device=device, dtype=dtype)
-        imag_data = tensor([0.0], device=device, dtype=dtype)
+        real_data = tensor(1.0, device=device, dtype=dtype)
+        imag_data = tensor(0.0, device=device, dtype=dtype)
         if batch_size is not None:
             KORNIA_CHECK(batch_size >= 1, msg="batch_size must be positive")
-            real_data = real_data[None].repeat(batch_size, 1)
-            imag_data = imag_data[None].repeat(batch_size, 1)
+            real_data = real_data.repeat(batch_size)
+            imag_data = imag_data.repeat(batch_size)
         return cls(complex(real_data, imag_data))
 
     def inverse(self) -> 'So2':
         """Returns the inverse transformation.
 
         Example:
-            >>> s = So2.identity(batch_size=1)
+            >>> s = So2.identity(1)
```

Comment:
```suggestion
            >>> s = So2.identity()
```

Reply:
for some reason, __repr__ doesn't want to print out the full string 
`Parameter containing:
 tensor(1.+0.j, requires_grad=True)` when unbatched instead only prints `(1+0j)`.
 Prints the full string when i add in .z